minio/minio
# Stop async listing earlier

## Description

Stop async listing if we have not heard back from the client for 3 minutes.

This will stop spending resources on async listings when they are unlikely to get used. If client returns a new listing will be started on the second request.

Stop saving cache metadata to disk. It is cleared on restarts anyway. Removes all load/save functionality.

## Motivation and Context

Listing requests may be abandoned by the caller. React faster to that and stop the async listing to not waste resources.

## How to test this PR?

Start big listing, stop client after a few roundtrips. Observe listings being aborted.

## Types of changes
